src/controller/organization.controller.ts

```diff
@@ -0,0 +1,124 @@
+import { Controller, Get, Post, Put, Delete, Body, Param, UseGuards, Req, Res, HttpException, HttpStatus } from '@nestjs/common';
+import { OrganizationService } from 'src/organizations/organization.service';
+import { CreateOrganizationDto } from 'src/dto/create-organization.dto';
+import { UpdateOrganizationDto } from 'src/dto/update-organization.dto';
+import { JwtAuthGuard } from 'src/auth/jwt-auth.guard';
+import { check } from 'prettier';
+import { checkUser, hasScope } from 'src/utils/common.utils';
+import { Organization } from 'src/schema/organization.schema';
+
+
+@Controller('organization')
+export class OrganizationController {
+    constructor(
+        private readonly organizationService: OrganizationService
+    ) { }
+
+
+    // brainlife admin can access this and get all organizations
+    // if not bl admin then only get the organizations that the user is a member of
+    // as a admin or as a user
+    @Get()
+    @UseGuards(JwtAuthGuard)
+    async findAll(@Req() req, @Res() res) {
+        let where = {};
+        if (req.query.find) where = JSON.parse(req.query.find);
+
+        if (!hasScope(req.user, 'admin')) {
+            where['roles.members'] = req.user.sub;
+        }
+
+        const limit = req.query.limit || 0;
+        const skip = req.query.skip || 0;
+        const select =
+            req.query.select ||
+            'owner name created modified roles removed';
+        return res.json(
```

Comment:
Note for myself : Write test and fix

Reply:
what's the fix it requires?

Reply:
I want to test if it will handle complex mongo queries specifically about members in admin and users. I'm just skeptical about it.

Reply:
ok-- if you intend to do this on this PR, better do it before I review it.
if it is for another PR, better keep the note to yourself on code as TODO or somewhere else. it will certainly be lost here.